server/server/src/main.rs

```diff
@@ -34,6 +36,19 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
         files_db_client,
     });
 
+    {
+        let mut con = futures::executor::block_on(server_state.index_db_pool.get()).unwrap();
+        let feature_flags = [FEATURE_FLAG_NEW_ACCOUNTS_FIELD];
+
+        for flag in feature_flags {
+            if !futures::executor::block_on(con.hexists::<_, _, bool>(FEATURE_FLAGS_KEY, flag))
+                .unwrap()
+            {
+                futures::executor::block_on(con.hset(FEATURE_FLAGS_KEY, flag, true)).unwrap()
+            }
+        }
+    }
+
```

Comment:
Apart from the `block_on` weirdness we just discussed in slack, I view this as some random logic put in our main file. Observe there's an attempt here to move things into fns or have 1-liners.

I do think there will be more things we do at the moment of startup, maybe we want to have a file that has a function called `pre_request_work`, and in there just goes through any startup validation we may want to perform (of which this is the only one).

I think we're going to have similar stuff for stuff our server does in the background (calculate metrics, for instance).

But I think we could also get away from all these stray calls, maybe we don't even need anything in `keys.rs`. Maybe you could (like `content_cache.rs` expose a `feature_flags.rs` which neately encapsulates all this business. I can picture 3 fns:

+ `initialize_flags`
+ `new_accounts_enabled -> bool`
+ `set_new_account_status(bool)`

Yeah I think that's going to solve a lot of the above feedback in a more neat way